config/bigquery.yml

```diff
@@ -1,6 +1,6 @@
 config:
   project:                mlab-speedup-integration
-  dataset:                MLab_Louisville_Data
+  dataset:                MLab_America_Data
```

Comment:
We probably don't want to adjust this until the dataset is made, otherwise, we can't even try to load the Louisville data.